packages/components/src/font-size-picker/index.js

```diff
@@ -18,11 +18,32 @@ import RangeControl from '../range-control';
 import { default as UnitControl, useCustomUnits } from '../unit-control';
 import CustomSelectControl from '../custom-select-control';
 import { VisuallyHidden } from '../visually-hidden';
+import CheckboxControl from '../checkbox-control';
```

Comment:
I'd like to try making this a default for font-sizes above a given threshold and maybe opt-in for em/rem.

Reply:
There's a lot of variation depending on device size, but generally I've found that font sizes that are `64px` and above tend to break poorly on small screens. Maybe we use that as a threshold and see how it feels?